private/rules/v2_lock_file.bzl

```diff
@@ -189,4 +193,5 @@ v2_lock_file = struct(
     get_artifacts = _get_artifacts,
     get_netrc_entries = _get_netrc_entries,
     render_lock_file = _render_lock_file,
+    has_m2local = _has_m2local,
```

Comment:
You'll need to make a matching change in the `v1_lock_file.bzl` file too

Reply:
Done.